rllib/examples/re3_exploration.py

```diff
@@ -0,0 +1,50 @@
+"""Examples demonstrating the usage of RE3 exploration strategy.
+
+To use RE3 user will need to patch the callbacks with RE3 specific callbacks
+and set the `exploration_config` with RE3 configs.
+
+```Python
+
+class RE3Callbacks(RE3UpdateCallbacks, config["callbacks"]):
+    pass
+
+config["callbacks"] = partial(RE3Callbacks, embeds_dim=128,
+    beta_schedule="linear_decay", k_nn=50)
+config["exploration_config"] = {"type": "RE3"}
+"""
+from functools import partial
+import ray
+from ray.rllib.agents import sac
+from ray.rllib.utils.exploration.random_encoder import RE3UpdateCallbacks
+
+if __name__ == "__main__":
+    ray.init()
+
+    config = sac.DEFAULT_CONFIG.copy()
+    beta_schedule = "linear_decay"
+
+    # Patch user given callbacks with RE3 callbacks for using RE3 exploration
+    # strategy
+    class RE3Callbacks(RE3UpdateCallbacks, config["callbacks"]):
```

Comment:
sac.DEFAULT_CONFIG doesn't specify callbacks?
it seems like you can simply do:
```
config["callbacks"] = partial(RE3Callbacks, embeds_dim=128,
    beta_schedule="linear_decay", k_nn=50)
```
here.

Reply:
This was used for demonstration purpose, in case user wants to know how it works when callbacks is provided in configs.